components/lib/crash/src/main/java/mozilla/components/lib/crash/prompt/CrashReporterActivity.kt

```diff
@@ -70,12 +77,12 @@ class CrashReporterActivity : AppCompatActivity() {
             return
         }
 
-        launch(UI) {
-            launch(CommonPool) {
-                crashReporter.submitReport(crash)
-            }.join()
+        GlobalScope.launch {
```

Comment:
Did some reading on the new APIs. Seems like we should define a CoroutineScope now? Basically implement `CoroutineScope` and override `coroutineContext`. So that execution lifecycle can be tied to this activity: 
https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-coroutine-scope/index.html
https://medium.com/@elizarov/structured-concurrency-722d765aa952

Use GlobalScope seems to be discouraged....

Reply:
Yeah, watched the new `Coroutines` talk of this year's KotlinConf. This makes a lot of sense.